docs/style/js/catalog/common.js

```diff
@@ -25,7 +25,7 @@ var brooklyn = (function ($, _) {
         },
 
         entityCard: _.template(
-            "<a class='plain' data-type='<%= type %>' href='entities/<%= type %>.html'>" +
+            "<a class='plain' data-type='<%= type %>' href='brooklyn-node.html#entities/<%= type %>'>" +
```

Comment:
Yes you are right about the SEO. Generally it is not good to have Ajax requests in the first place.
Although it is not recognized well in google the catalog classes are listed in google https://www.google.bg/?q=brooklynnode
In order to mimic the current SEO capabilities of the website we have to support https://developers.google.com/webmasters/ajax-crawling/docs/getting-started
_escaped_fragment_ get parameter on the server.
 I will research this more if it is possible to use this get_parameter without implementing it on the server and rather use the capabilities of the modern browsers like pushState. 
The other option is to autogenerate sitemap.xml